ts/utils/hooks/bottomSheet.tsx

```diff
@@ -168,3 +175,62 @@ export const useIOBottomSheetModal = (
   );
   return { present, dismiss: dismissAll, bottomSheet };
 };
+
+/**
+ * Hook to generate a bottomSheet with a title, snapPoint and a component, that autosizes to the height of its content
+ * @param bottomSheetOptions
+ * @see {BottomSheetOptions}
+ * @param bottomPadding the bottom padding of the bottom sheet, defaul is 0
+ */
+export const useIOBottomSheetAutoresizableModal = (
+  {
+    component,
+    title,
+    snapPoint: originSnap,
+    footer,
+    onDismiss
+  }: BottomSheetOptions,
+  bottomPadding: number = 0
+) => {
+  const [snapPoint, setSnapPoint] = React.useState<number>(originSnap[0]);
```

Comment:
Then to reach the desiderated behavior (omitting the snapPoint param from the autoresizable hook), for this line of code since the `@gorhom/bottom-sheet` library requires a snapPoint different from zero and a positive number, I suggest to add a constant anywhere in the file like the follow:

```typescript
// Snap point used as default for autoresizable bottom sheets, it is not zero since the bottom sheet library throws an error
const DEFAULT_AUTORESIZABLE_SNAP_POINT = 1;

```

And then this line of code becomes:

```suggestion
  const [snapPoint, setSnapPoint] = React.useState<number>(DEFAULT_AUTORESIZABLE_SNAP_POINT);
```